code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1814,6 +1814,36 @@
 			H.facial_hair_style = "Beard (Very Long)"
 			H.update_hair()
 
+/datum/reagent/barbers_afro_mania
+	name = "Barber's Afro Mania"
+	description = "Unleash the funk within you."
+	reagent_state = LIQUID
+	color = "#FF8800"
+	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY | CHEMICAL_GOAL_BOTANIST_HARVEST
+	taste_description = "funky sugar"
+
+/datum/reagent/barbers_afro_mania/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(method == TOUCH || method == VAPOR)
+		if(M && ishuman(M))
```

Comment:
This is wrong. `ishuman(mob)` counts every humankind species like lizard, moth.
You should use `ishumanbasic(mob)`, and possibly `iscatperson(mob)` too.

Reply:
Since those don't have hair it does nothing to them. Barbers aids are coded like that, I'm guessing so it automatically targets everything that can use hair (humans,ethereals, and felinids)

Reply:
I am keeping it like that if that's okay